packages/app-bridge-theme/src/types/Guideline.ts

```diff
@@ -56,53 +63,59 @@ export interface GuidelineDocumentLink {
     parentId(): Nullable<number>;
     type: 'document-link';
 }
+export interface DocumentLinkNavigationItem extends DocumentLinkBase {}
```

Comment:
IMO we don’t need the `DocumentLinkBase` for now. Once we need a `DocumentLink` we can easily add it. Wdyt?

Reply:
same thing that `documentGroup`

Reply:
wait, i got what you said :)